src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java

```diff
@@ -108,8 +108,8 @@
             //"org.jitsi.impl.neomedia.codec.video.h263p.Packetizer",
             // Adaptive Multi-Rate Wideband (AMR-WB)
             // "org.jitsi.impl.neomedia.codec.audio.amrwb.DePacketizer",
-            "org.jitsi.impl.neomedia.codec.audio.amrwb.JNIDecoder",
-            "org.jitsi.impl.neomedia.codec.audio.amrwb.JNIEncoder",
+            //"org.jitsi.impl.neomedia.codec.audio.amrwb.JNIDecoder",
+            //"org.jitsi.impl.neomedia.codec.audio.amrwb.JNIEncoder",
```

Comment:
I can't remember doing anything with AMRWB. Is it necessary to disable that? Or is/was it broken anyway?

Reply:
We no longer add it to the ffmpeg (I think the license is not compat), so I removed it as it will not work anyway.

Reply:
opencore-amr is Apache 2 licensed on SourceForge. Anyway, I don‘t think it is a necessary codec.